Packs/PAN-OS/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "PAN-OS by Palo Alto Networks",
     "description": "Manage Palo Alto Networks Firewall and Panorama. Use this pack to manage Prisma Access through Panorama. For more information see Panorama documentation.",
     "support": "xsoar",
-    "currentVersion": "2.1.3",
+    "currentVersion": "2.1.5",
```

Comment:
Why the double version bump? (2.1.3 --> 2.1.5)?

Reply:
It got off somehow. I will update after pulling & merging from master.